src/styles/index.css

```diff
@@ -545,12 +545,6 @@ body {
   color: var(--p-color-text-default)
 }
 
-* {
```

Comment:
@znicholasbrown I assume this wildcard transition was added to animate the light/dark modes. Lmk if you feel strongly about it, but I felt it was a bit overreaching for an affect that is pretty noticeable and thus probably doesn't warrant the transition.

Reply:
It's fine to remove